shared/js/toc.jsx

```diff
@@ -125,27 +128,32 @@ loop.shared.toc = (function(mozL10n) {
               label={this.state.roomName}
               onEditionComplete={this.exitEditMode} />
           </div>
-          <RoomPresenceView />
+          <RoomPresenceView
+            participantStore={this.props.participantStore} />
           <RoomActionsView
             addUrlTile={this.props.addUrlTile} />
         </div>
       );
     }
   });
 
-  // XXX akita: Make this work
   var RoomPresenceView = React.createClass({
-    propTypes: {},
+    propTypes: {
+      participantStore: React.PropTypes.instanceOf(loop.store.ParticipantStore).isRequired
+    },
 
     render: function() {
       return (
         <div className="room-active-users">
-          <div className="room-user" data-name="Pau Masiá">
-            <span>{'P'}</span>
-          </div>
-          <div className="room-user" data-name="Manu">
-            <span>{'M'}</span>
-          </div>
+          {
+            this.props.participantStore.getOnlineParticipants().map(function(participant, index) {
+              return (
+                <div className="room-user" data-name={participant[1].participantName} key={index}>
+                  <span>{participant[1].participantName.substring(0, 1).toUpperCase()}</span>
```

Comment:
@Mardak  I think the timestaps calculated in the store are wrong. Let's imagine I've updated my ping with Date.now() - 1000 so the localPingTime=1000 then I invoke getOnlineParticipants and Date.now() - participant.localPingTime <= 60000 is false because of that, I'm not online